### PR TITLE
StylableText component

### PR DESCRIPTION
**Pull request by amiryonatan**

### CODE REVIEW

File: packages/wix-ui-core/src/components/StylableText/Text.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      expect(driver.exists()).toBeTruthy();
+    });
+
+    it('should have ellipsis', () => {
+      const driver = createDriver(<Text ellipsis>Hello World</Text>);
+      expect(driver.isEllipsis()).toBeTruthy();
+    });
+
+    it('should not have title attribute by default', () => {
+      const driver = createDriver(<Text ellipsis>Hello World</Text>);
```
Comment (shlomitc) [inline packages/wix-ui-core/src/components/StylableText/Text.spec.tsx]: it looks exactly like the next text. what is the difference?

File: packages/wix-ui-core/src/components/StylableText/Text.st.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+}
+
+/* ellipsis state */
+.root:ellipsis {
+    text-overflow: ellipsis;
+    display: inline-block;
+    width: 100%;
+    overflow: hidden;
+    white-space: nowrap;
+}
```
Comment (shlomitc) [inline packages/wix-ui-core/src/components/StylableText/Text.st.css]: missing EOL

File: packages/wix-ui-core/stories/Text/index.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  category: 'Components',
+  name: 'Text',
+  storyName: 'StylableText',
+  component: Text,
+  componentProps: (setState) => ({
+    'data-hook': 'storybook-text',
+    ellipsis: true,
+    children: 'hello'
+  }),
+  exampleProps: {
```
Comment (shlomitc) [inline packages/wix-ui-core/stories/Text/index.ts]: remove

File: packages/wix-ui-core/stories/index.tsx [hunk 4]
```diff
@@ -24,6 +24,7 @@ const dropdownOptions =
 require('./Badge').story();
 require('./Button').story();
 require('./Input').story();
+require('./Text').story();
```
Comment (shlomitc) [inline packages/wix-ui-core/stories/index.tsx]: `StylableText`

File: packages/wix-ui-core/src/components/StylableText/Text.spec.tsx (hunk 1 above)
Comment (shirlev-wix) [reply packages/wix-ui-core/src/components/StylableText/Text.spec.tsx]: I guess nothing. Was copied from the old text tests.. Removing..

Comment (shirlev-wix) [reply packages/wix-ui-core/src/components/StylableText/Text.spec.tsx]: Actually I think this test is needed if i'm reading the description. The test just doesn't match the title.

File: packages/wix-ui-core/src/components/StylableText/Text.st.css (hunk 2 above)
Comment (shirlev-wix) [reply packages/wix-ui-core/src/components/StylableText/Text.st.css]: EOF?

Comment (shlomitc) [reply packages/wix-ui-core/src/components/StylableText/Text.st.css]: yes

**Pull request merged**